### ISSUE TITLE
Incorrect type hint for "bucket_class" in "Limiter"

### ISSUE THREAD

**Issue by ryanzlu** [vutran1710/PyrateLimiter#30]
The `bucket_class` in `Limiter` requires a type as input instead of an object instance of type `AbstractBucket`. 

Current:
```
class Limiter:
    def __init__(
        self,
        *rates: RequestRate,
        bucket_class: AbstractBucket = MemoryQueueBucket,
        bucket_kwargs=None,
    ):

```

Should change to:
```
from typing import Dict, Union, Type

class Limiter:
    def __init__(
        self,
        *rates: RequestRate,
        bucket_class: Type[AbstractBucket] = MemoryQueueBucket,
        bucket_kwargs=None,
    ):
```

**Comment by vutran1710** [comment]
Interesting! Thanks for pointing it out! Will be fixed soon.

**Comment by vutran1710** [comment]
solved in `2.3.2`
